Packs/GSuiteAdmin/Integrations/GSuiteAdmin/GSuiteAdmin.yml

```
@@ -2229,7 +2228,629 @@ script:
+        name: customer_id
+        required: true
+      - auto: PREDEFINED
+        description: In case of Group use “POST…groups:batchModify”. In case of OrgUnit use “POST…orgunits:batchModify”.
+        name: target_type
+        required: true
+        predefined:
+          - Group
+          - OrgUnit
+      - description: |-
+          String as JSON representation of the ModifyGroupPolicyRequest
+          {
+              "requests": [
+                  {
+                      "policyTargetKey": {
+                          "targetResource": "orgunits/03ph8a2z1kjba6k",
+                          "additionalTargetKeys": {
+                              "app_id": "chrome:jldhpllghnbhlbpcmnajkpdmadaolakh"
+                          }
+                      },
+                      "policyValue": {
+                          "policySchema": "chrome.users.apps.InstallType",
+                          "value": {
+                              "appInstallType": "BLOCKED"
+                          }
+                      },
+                      "updateMask": "appInstallType"
+                  }
+             ]
+          }
+        name: policy_raw_json
+      - description: Entry Id of the file that contains JSON representation of the ModifyGroupPolicyRequest.
+        name: policy_field_json_entry_id
+      - description: |-
+          The target resource on which this policy is applied. The following resources are supported:
+
+          Organizational Unit ("orgunits/{orgunit_id}")
+          Group ("groups/{groupId}").
+
+        name: target_resource
+      - description: |-
+          Map containing the additional target key name and value pairs used to further identify the target of the policy.
+
+          A string containing a list of "key": "value" pairs. Example: ""name": "wrench", "mass": "1.3kg", "count": "3"".
+
+        name: additional_target_keys
+      - description: "The fully qualified name of the policy schema that is being inherited."
+        name: policy_schema
+      - description: "The value of the policy that is compatible with the schema that it is associated with Struct format."
+        name: policy_value
+      - description: |-
+          Policy fields to update. Only fields in this mask will be updated; other fields in policyValue will be ignored (even if they have values). If a field is in this list it must have a value in 'policyValue'.
+          This is a comma-separated list of fully qualified names of fields. Example: "user.displayName,photo".
+          Field mask format.
+        name: update_mask
+    description:  Delete multiple policy values that are applied to a specific group. All targets must have the same target format. That is to say that they must point to the same target resource and must have the same keys specified in additionalTargetKeyNames, though the values for those keys may be different. On failure the request will return the error details as part of the google.rpc.Status.
```

```suggestion
    description:  Modifies multiple policy values that are applied to a specific group. All targets must have the same target format. That is to say that they must point to the same target resource and must have the same keys specified in additionalTargetKeyNames, though the values for those keys may be different. On failure the request will return the error details as part of the google.rpc.Status.
```
don't forget to update the readme